openedx/core/djangoapps/course_date_signals/models.py

```diff
@@ -0,0 +1,11 @@
+from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
+
+
+class SelfPacedRelativeDatesConfig(StackedConfigurationModel):
+    """
+    Configuration to manage the SelfPacedRelativeDates settings.
+
+    .. no_pii:
+    """
+
+    STACKABLE_FIELDS = ('enabled')
```

Comment:
This is already the default in the parent class, except you're also switching it to a string from a tuple.

Reply:
Cool, I'll get rid of this line, then.